data_retriever/migration_event_queue.py

```
@@ -0,0 +1,204 @@
+
+    def push(self, event, is_rollback=False):
+        """
+        Push an event to the queue
+        Args:
+            event (VMMigrationEvent | VMShutdownEvent | ServerShutdownEvent): The event to push to the queue
+            is_rollback (bool): Whether the event occured during migration or rollback. Default to False for migration
+        Raises:
+            EventQueueException: If push could not be performed
+        """
+        if not self._conn or not self._cursor:
+            raise EventQueueException(f"Postgres connection not established")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)